demo/server/src/modules/index.ts

```diff
@@ -20,13 +23,27 @@ function loggerSetup() {
 }
 
 function securitySetup(app: express.Express) {
-  setupPassportSecurity(app);
+  if (useKeycloak) {
+    // user keycloak authentication
+    setupKeycloakSecurity(app);
+  } else if (process.env.NODE_ENV === 'production') {
```

Comment:
Demo application has profile based configurations. We should not use `process.env.NODE_ENV === 'production'` directly. I'm running my node in production mode all the time because of some reasons. My vote will be to not imply this in code and just document this carefully as note.